distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -76,6 +77,20 @@ type Type
         c = self.value
         get_short_type_name c
 
+    ## ADVANCED
+       GROUP Metadata
+
+       Finds type specified by fully qualified name.
+
+       Arguments:
+       - qualified_name: fully qualified name.
+    find : Text -> Type ! Not_Found
+    find qualified_name =
```

Comment:
If I don't have _any_ imports for `Standard.Table` in my project and I do `Meta.find "Standard.Table.Data.Table.Table"`, what will happen? Will the `Standard.Table` get loaded? Or are only the currently loaded modules searched?

---

IMO both approaches are fine and I'd lean towards _not_ triggering an import by this method. But I think it would be good to clarify this in the documentation comment for `find`.

Reply:
No. We only search already loaded types. Clarified in f5921e1